IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_MechanicalFastenerOverheadContactSystemRigidSupport/DocPropertySet.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_MechanicalFastenerOverheadContactSystemRigidSupport" UniqueId="37ce8447-2a11-e7c4-7e22-82827dd949f7" ApplicableType="IfcMechanicalFastener" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
@fhulin Slightly Confused, we have 2 property sets which have the same properties:

- Pset_MechanicalFastenerOverheadContactSystemRigidSupport
- Pset_OCSRigidSupport

1. Which one is correct? I assume one is a mistake. please then apply the relevent suggestions.
2. rename  Pset_OCSRigidSupport --> PSet_MemberTypeOCSRigidSupport